ScanCommand/src/test/java/io/ballerina/scan/TestScanCmd.java

```diff
@@ -0,0 +1,84 @@
+/*
+ *
+ *  * Copyright (c) 2024, WSO2 LLC. (https://www.wso2.com).
+ *  *
+ *  * WSO2 LLC. licenses this file to you under the Apache License,
+ *  * Version 2.0 (the "License"); you may not use this file except
+ *  * in compliance with the License.
+ *  * You may obtain a copy of the License at
+ *  *
+ *  *   http://www.apache.org/licenses/LICENSE-2.0
+ *  *
+ *  * Unless required by applicable law or agreed to in writing,
+ *  * software distributed under the License is distributed on an
+ *  * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ *  * KIND, either express or implied. See the License for the
+ *  * specific language governing permissions and limitations
+ *  * under the License.
+ *
+ */
+
+package io.ballerina.scan;
+
+import org.apache.commons.lang3.SystemUtils;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.nio.charset.StandardCharsets;
+import java.util.ArrayList;
+import java.util.List;
+
+public class TestScanCmd {
+    @Test
+    void test_scan_command() {
```

Comment:
This test is fine for now, since we haven't added the test command logic yet. But let's have some more complex scenarios in tests once we add the core logic.